datastore/google/cloud/datastore/transaction.py

```diff
@@ -152,13 +153,22 @@ def Entity(*args, **kwargs):
 
     :type client: :class:`google.cloud.datastore.client.Client`
     :param client: the client used to connect to datastore.
+
+    :type read_only: bool
+    :param read_only: indicates the transaction is read only.
     """
 
     _status = None
 
-    def __init__(self, client):
+    def __init__(self, client, read_only=False):
         super(Transaction, self).__init__(client)
         self._id = None
+        if not read_only:
```

Comment:
This is definitely a nit-pick, but I would prefer you switch the order of the branches and drop the negation (it is unnecessary "work" to negate the boolean).

Reply:
I was assuming that read_only is False by default, so I made that the first option.  I can change it if you want.

Reply:
I want